src/main/resources/application.properties

```diff
@@ -26,6 +26,8 @@ server.address=localhost
 server.port=8080
 hostname=localhost
 
+ctt.studyRooms=011;012;013;014;015;111;112;113;114;115;206;207;208;209;210
```

Comment:
Mir fällt gerade auf, das in der Production-DB die Räume im EG ohne führende Null gespeichert sind. Bitte nochmal prüfen, ob das eine Rolle spielt und ggf. hier die Nullen entfernen.

Reply:
Ja das stimmt, habe es korrigiert.